packages/maskbook/src/plugins/COTM/assets/Lisa.ts

```diff
@@ -0,0 +1 @@
Ja5X4EQKfywmpBJlCvGNtsl6GfFliUvjIRe8VncT65NeS95EzSyN0l/XCr1/wxK/csl6W8EWmksDV/u85sIbyrPH3mssHE8AvPkWgUvv85zJZNvr+9m9/kAD2p9+/MVu+xrz5u1+3vj+9EdD++tnHEszaM+9aP/CYNL+FBzjkfvlLCWZfff7GbHmH6e3nn0g4a32rtj/A5fPF5WVWkzn4RiWd5PDbaaxerud1SfmmKEeEBgthBfNZL5RoDiOq0XnIUrHt/2zwrorQl4rajTxmaOocJQhsKUJuUyDChoUQ5wY7h9b2fwklqrOpPGaJVpk/OiGMoABw0TBZGv6Y54KLvoLrX+XLlvnCO1oINl8Sra+cW6x7rz9Dq/kSdLm4e+ELuC4nc2C+7COEKeUFATfiC3PQ+hxpab6QIsMC3Knfi3dfp4PZldlSUdeM09rib/oxYfp8QJOuO6LBEN0XJX5+kkr04ZR/wVkyOHcxScZYFfsGHi2ZBQ3MopN7FpOm7750H3HR27SGQCUt+/qeZZ3m84GGMGZ4hMq33n1xp8fdh9LFuIB7m0wAjxzReI7zfaWVsZvvIwIluvrRS/OF0oW8E6UCpYApfLkY8wV5ocoEbWxEe+50MsE2aS6S/FHP+DFflso3morD27pn+OXjAV8yLj4tKxEQ0MtKEsycfO4iSlXYdstLw6QoJaMkrzT3OWIEgiCyd1hmHAn+Yqwuu9ZfcQn+/2Z4qSULrvMprfQNcCHt9/ArwRdar2Q6SvAr4BFSurEsMhaeM7QbHtir6KMZHpiMK8PzSfTORlz0rcSDQmvFtT2iNb8XKtQeF5Ew8A81LL98788X1cHcmeCSIIZGLGjFyZFLj8X6KOFi3GZadfGnbN0LzHdbEdjAejhnHbdx7rtTKRYENDZAD09fqggAzKXG8izDe23OXi8XrkiQ9xF7LI1N60W4WP+DtVn4wkHLQV7mAJPmTV8BgrWHKp7mse1/r9u41x89tS8ZDeFMfQjxhgcG4RP4KKuF6RKfMj7DMRbeceDhf9G/az5fxnPABcTcskYm2eFPdUEnR3DGXeSx/I1JMOG6Vp8jDw0PLlrBdcblkGgNXJnWjMtRhdB6NaF1WXFBD2MrrQnX/fmynM8XhK6ZfDGuypfzQmvBNeaLSlIleJugV1NaZVpOe9amztvJHCzC7214PijB6E/or3ff43T3jc94r+28Z7l3gK93X727tOe3N919k3t2dE9DV8XVmseWu6/e04FrvGd9z0Ywkn/A0jbdR+Xuu0TYFq56lg344rFDvmhsofViW7RW+eNB4ctyNl/WZ/xA/ih8ub+sFLUy268C2ZztGOoaXFyK/UXN4masCTMB/Fq3I3UsItR8qIfTWCTtRWtIkBWX8WvsEvgJLpiymNTqlF0a+75xFVrjgKm4eG/8AzbhavDgymOBv5zBF+O6zrikhn40xQX8w8IXTGJ8dnzBX41Fn+BZ8Iz1HJovvNdlpotfeq0/4cI3osCzqTbiAv69NJaNyFgdXBFlifYMx/38Xqib8WOL95K9nrGuN2ZcO9v0p3qprLdTTF30xcHLukJgy+/K2tRh0gtrmNM4kNPYs4caO16blS/jtTnmy3htbpV9yHecZuH5hvWSNzjmBfxf5Jgaghm+aa2f7xa2Xv/0fZkwu7Gvm2D+umm6XqO6b2NibBPWZBqN5yK8YVrl0/ONdoEUKlSUMA+4NPGPpB/NucfubItWaRxjbKPrRgEoIVRovpm3LACxl4AJvnJ508+lHvD4dzBneb0wjjnv4XkeedkKrpOHFRdaJvpvelyYJPFldJ9xQVelVb5h/T4EvuJSolVoTQKQxl4moURjFzNwwQ/gM1+MK8OfC77hKnypuICXsJNxKY+g+2IOJvtwud6z1z1fNIfQkcaibdWe7XCBXybMdCEu2F/5fOBCNa5yd3HO3OPum3fGu/ZinA+cpeWM59zmnB6c8XPvPnCV8+V63t3H2CkuziLOshl3X4IvMkE9y+bJBNxbjM988d1VaIVneW1V+aPyxcKW+TJf/phxxlsmGPFFglnyuzqwGS0nXbUpM6nccMSnD8R2tOMTWDaczZs0lzTAfuu+fUWhQJRx8akFNsBlRz2PhXm0vQ/t7BiRXyR/zXZfSbAVV64EfyoVa8qUTGmOe/DFxU4TrpUOKnzgPNZRJA1n4Dq0mhuz68eyv/Ppi/mkUwefkhtMi5VxHmu+kFvGc2u+pKSIRyq3wUZLuMBN/3HnZH6mC5ToI/eR50Z8QfVsv6ePf4ow5Xxj9htr35+qKDmROaoowCdF3xXZi9o8/PGEq+GM5KrCJX8RmST4XKpcWDNVhD+ZSwchpLmPsSubYJ7IjEY70YFItJf7iPIarJchX5jLlKSYdxrwRWszrxf5pnwkPxQ+geUZOPp3jsheL8UJ9o7N3kyL183Zn89bzC0cHr9q5sc//k6tJfr9to39rglff/uPP7b2p+//9u9//P41pp0G/10zV37XTJy012tT5utm/nyNuZNGQAAHCibVP/yWZ+qT+eJybebN2/YMGiZQcN0SIPLbL1v76vsVfmccXmiBv/pCfY3eWwRB5qbBQD9t1Uzd0M/F2c/3pZz38TPqHddl2gq/KXJeqQZe9idzmaOUyHRXOZEQ0lImea+Xr8OhHkFEPkNhMoQW7fnwu+ITwQlak6M/vmGOaLR5kUCElUsXJVyMzbheDnBdrHE9D/jgC352gct8MX5a8KXDZRPvadDK50a+HAsXUa1R0aHDhXnSfL3QHMJ3+rIpExq7ShMraemYT48Nc3J7psfutr3lH2LAGd71FneVTDaZMk/CX5VcZjLxEumZEpZK43EUzvf4PI3vPnB9sFPOeBzYe18wn/E+H/i80twSzRk+bu07uBAy3n7GP/UZT/Rluk90lm3ZvMknuId3X5UJfJbZH0z30Qa+WHjqccXdh3mxygT9PQ2uIhNYQ3UUSXbx7xvcfdkcTXOpqSP84bP8MZ8v9Yw3X4Cz/AE
```

Comment:
Why not store it as a picture?

Reply:
The CORS strategy prohibits the image from `/public` dir.

Reply:
So you should use the `<Image />` element in our codebase. It supports workaround CORS